scripts/Muon/GUI/Common/corrections_tab_widget/background_corrections_view.py

```diff
@@ -266,15 +275,30 @@ def _on_item_clicked(self, item: QTableWidgetItem) -> None:
         self._selected_column = item.column()
         self._selected_value = item.text()
 
-    def _on_corrections_table_cell_changed(self, _: int, column: int) -> None:
+    def _on_corrections_table_cell_changed(self, row: int, column: int) -> None:
         """Handles when a cells value is changed in the corrections table."""
+        self._selected_row = row
+        self._selected_column = column
+
         self._format_selection()
 
-        if column == START_X_COLUMN_INDEX:
+        if column == USE_RAW_COLUMN_INDEX:
+            self._handle_use_raw_changed()
+        elif column == START_X_COLUMN_INDEX:
             self._handle_start_x_changed()
         elif column == END_X_COLUMN_INDEX:
             self._handle_end_x_changed()
 
+    def _create_bool_table_item(self, state: bool) -> QTableWidgetItem:
+        """Creates a check box table widget item with an initial boolean state."""
+        item = self._create_table_item("")
+        item.setFlags(Qt.ItemIsUserCheckable | Qt.ItemIsEnabled)
+        if state:
+            item.setCheckState(Qt.Checked)
+        else:
+            item.setCheckState(Qt.Unchecked)
+        return item
+
     def _create_double_table_item(self, value: float, editable: bool = True, enabled: bool = True) -> QTableWidgetItem:
```

Comment:
https://github.com/mantidproject/mantid/blob/master/scripts/Muon/GUI/Common/utilities/table_utils.py#L95

Reply:
Its not easy to reuse this here, but I will move my creation function to the table_utils file